### PR TITLE
ci: release and publish Tags

### PR DESCRIPTION
**Pull request by sladkoff**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ NODE_VERSION }}
+
+      - run: npm ci
+      - run: npm test
+      - id: publish
+        uses: JS-DevTools/npm-publish@v1
+        with:
+          token: ${{ secrets.NPM_TOKEN }}
```
Comment (sladkoff) [inline .github/workflows/release.yaml:27]: @NicolasBissig Please create a `NPM_TOKEN` secret 🙏

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          body_path: changelog_body.md
+          draft: true
+          prerelease: false
+          files: |
+            dist
+            README.md
+            CHANGELOG.md
+          fail_on_unmatched_files: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (sladkoff) [inline .github/workflows/release.yaml:49]: @NicolasBissig Please also create a `GITHUB_TOKEN` secret.

**Review by sladkoff — COMMENTED** [review]
@NicolasBissig This is not tested yet. If you provide the needed secrets I'd run some tests with some fake tags.

**Comment by NicolasBissig** [comment]
Thanks! I have set up the secrets

**Comment by sladkoff** [comment]
@NicolasBissig Alright, this seems to work.

On the latest tag `v0.1.2-1` the following stuff was done:

- Published to [npm](https://www.npmjs.com/package/azure-functions-decorators/v/0.1.2-1)
- Created a [gh release draft](https://github.com/NicolasBissig/azure-functions-decorators/releases/tag/untagged-6d7b0f4c30296fa11e14) which can be published manually. In theory, this should also prepare content from the `CHANGELOG.md` but I was too lazy to verify this.

**Comment by NicolasBissig** [comment]
Thanks a lot!

**Pull request merged**